code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -170,6 +172,21 @@
 				else
 					L.reagents.add_reagent("spidertoxin", 5)
 
+/mob/living/simple_animal/hostile/poison/bees/proc/make_opaque()
+	// If a bee attacks someone, make it very easy to hit for a while
+	last_attack = world.time
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
+
+/mob/living/simple_animal/hostile/poison/bees/Life(seconds, times_fired)
+	. = ..()
+	var/diff = world.time - last_attack
+	if(diff >= opacity_time)
+		mouse_opacity = initial(mouse_opacity)
+
+/mob/living/simple_animal/hostile/poison/bees/sentience_act()
+	. = ..()
+	make_opaque()
```

Comment:
This will be undone when enough time has passed, you'd need another variable or something for this case